docs/guide/testing-types.md

```diff
@@ -4,6 +4,18 @@ title: Testing Types | Guide
 
 # Testing Types
 
+::: tip Sample Project
+
+[GitHub](https://github.com/vitest-dev/vitest/tree/main/examples/typecheck) - [Play Online](https://stackblitz.com/fork/github/vitest-dev/vitest/tree/main/examples/typecheck?initialPath=__vitest__/)
+
+:::
+
+::: tip
+
+You can find a sample project in [Examples](/guide/#examples).
+
+:::
+
```

Comment:
I tested these two patterns and probably having direct links looks nicer. What do you think?

![image](https://github.com/vitest-dev/vitest/assets/4232207/a2431e1e-9fba-47a6-923b-6b06a8dfb537)

![image](https://github.com/vitest-dev/vitest/assets/4232207/e039e587-ddc2-4d4d-9b55-047e1323e850)

Reply:
I like the first pattern